components/StyledInputAmount.js

```diff
@@ -158,7 +158,8 @@ const StyledInputAmount = ({
   return (
     <StyledInputGroup
       maxWidth="10em"
-      step="0.01"
+      // Increase/Decrease the amount by $0.5 instead of $0.01 when using the arrows
+      step="0.5"
```

Comment:
@Betree it seems the `step` prop was already available. I didn't know such an attribute exists on [input fields](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/step)


https://user-images.githubusercontent.com/46647141/218070881-854d51ff-7215-4372-8fe8-8c36bb86f656.mov

Reply:
An issue with this is that it won't allow custom amounts like `20.45`
<img width="532" alt="Screenshot 2023-02-13 at 10 30 17 AM" src="https://user-images.githubusercontent.com/46647141/218374892-73db9f04-0090-446f-bdc2-3f8c89175688.png">

Reply:
Yes, that's why I mentionned in the issue:

> Make sure it is still possible to manually input any custom amount in the box

And suggested resolving this by:
> It would be great to have a solution that we could easily re-use, maybe a `arrowStep={0.5}` option on `StyledInputAmount`? The change is not requested to complete the bounty as it comes as a later specification. We can do it as a follow-up.